docs/storing-querying/schemas-in-warehouse/index.md

```diff
@@ -332,7 +351,24 @@ Note that this behavior was introduced in RDB Loader 5.3.0.
 
   </TabItem>
   <TabItem value="bigquery" label="BigQuery">
+    <Tabs groupId="biquery-loader-version" queryString lazy>
+        <TabItem value="v2" label="Version 2.x" default>
+
+Because the column name for the self-describing event or entity includes the major schema version, each major version of a schema gets a new column:
+
+| Schema | Resulting column |
+|---|---|
+| `com.example/button_press/jsonschema/1-0-0` | `unstruct_event_com_example_button_press_1` |
+| `com.example/button_press/jsonschema/1-2-0` | `unstruct_event_com_example_button_press_1` |
+| `com.example/button_press/jsonschema/2-0-0` | `unstruct_event_com_example_button_press_2` |
 
+When you evolve your schema within the same major version, (non-destructive) changes are applied to the existing column automatically. For example, if you add a new optional field in the schema, a new optional field will be added to the `RECORD`.
+
+:::info Breaking changes
+<ParquetRecoveryColumns/>
+:::
+   </TabItem>
+   <TabItem value="v1" label="Version 1.x">
```

Comment:
Could you adjust the text below? (See emphasized parts)

---

While our recommendation is to use major schema versions to indicate breaking changes (e.g. changing a type of a field from a `string` to a `number`), this is not particularly relevant with BigQuery **Loader version 1.x**. Indeed, each schema version gets its own column, so there is no difference between major and minor versions. That said,  sticking to our recommendation is a good idea:
 * Breaking changes might affect downstream consumers of the data, even if they don’t affect BigQuery
 * **Version 2 of the loader has stricter behavior that matches our loaders for other warehouses and lakes**